corehq/apps/userreports/tests/test_view.py

```diff
@@ -133,6 +133,9 @@ def _delete_everything(cls):
     @classmethod
     def tearDownClass(cls):
         cls._delete_everything()
+        # todo: understand why this is necessary. the view call uses the session and the
+        # signal doesn't fire to kill it.
+        Session.remove()
```

Comment:
Is it not because you've removed `connection_manager.dispose_all()` from other test tear down methods?  When you say the signal doesn't fire, do you mean the  `request_finished` signal? Perhaps its this test isn't using the request cycle to test the view?

I don't thing `connection_manager.dispose_all()` should be required in tearDown methods but it might be necessary to have `Session.remove()`